scripts/ansible/roles/linux/validation/tasks/slave-validation.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+---
+- name: Check for existing required binaries
+  stat:
+    path: "{{ item }}"
+  with_items: "{{ validation_binaries }}"
+  register: binary
+  failed_when: binary.stat.executable == False
+
+- name: Check for existing required directories
+  stat:
+    path: "{{ item }}"
+  with_items: "{{ validation_directories }}"
+  register: directory
+  failed_when: directory.stat.isdir == False
+
+- name: Check for existing required files
+  stat:
+    path: "{{ item }}"
+  with_items: "{{ validation_files }}"
+  register: file
+  failed_when: file.stat.exists == False
+
+- name: Cmake version check
+  command: "cmake --version"
+  register: cmake_check_version
+  failed_when: not(cmake_check_version.stdout.find(cmake_target_version) != -1)
```

Comment:
Instead of using double negation:
```
not(ocaml_check_version.stdout.find(ocaml_target_version) != -1)
```
I think it is easier to use simply:
```
ocaml_check_version.stdout.find(ocaml_target_version) == -1
```

This applies to every check below.